Containerfile

```diff
@@ -27,10 +23,6 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -o manager main.go
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
 WORKDIR /
 COPY --from=builder /workspace/manager .
-COPY --from=builder /workspace/metrics_dashboards/ovms-metrics.json .
```

Comment:
this is not for this PR but just for your information, please remove this from midstream as well